individual-reflection.md

```diff
@@ -8,65 +8,86 @@ Answer the following comprehension questions **within this file.** Write your an
 
-      # Copy and paste your answer above this comment
-      ```
-1. What does the program do if the response comes back with a status code of 200?
-    - Answer: 
-1. What does the program do if the response does not come back with a status code of 200?
-    - Answer: 
+          # Copy and paste your answer above this comment
+
+1.  What does the program do if the response comes back with a status code of 200?
+    - Answer: The request will be valid and will return the response
+1.  What does the program do if the response does not come back with a status code of 200?
+    - Answer: if the response does not comeback with true or 200 there Slack Error will raise an error
 
 ### `POST` Request Review
 
 If your project does not make a POST request, read through Wave 3 on the original Slack CLI, and research and answer questions 1, 2, 3, 4, 6, and 7.
 
-1. Describe a POST request that your project makes, and the high-level description of what it does
-    - Answer:
-1. What is the verb of this request?
-    - Answer:
-1. What is the path (or the URL, or endpoint) of this request?
-    - Answer:
-1. What are the query params (the additional data sent with the request, besides the verb and the path)?
-    - Answer: 
-1. What is the syntax used to make this request? (Copy and paste a code snippet here)
+1.  Describe a POST request that your project makes, and the high-level description of what it does
+    - Answer: The post will update the data as it is posting to a channel or a user.
+1.  What is the verb of this request?
+    - Answer: POST
+1.  What is the path (or the URL, or endpoint) of this request?
+    - Answer: url = "https://slack.com/api/chat.postMessage?token=#{SLACK_TOKEN}&channel=#{@slack_id}&text=#{message}"
+1.  What are the query params (the additional data sent with the request, besides the verb and the path)?
+    - Answer: token, user/channel or text
+1.  What is the syntax used to make this request? (Copy and paste a code snippet here)
+
     - Answer:
       ```ruby
       # Copy and paste your answer below this comment
-
-      # Copy and paste your answer above this comment
+        def send_message(message)
+      url = "https://slack.com/api/chat.postMessage?token=#{SLACK_TOKEN}&channel=#{@slack_id}&text=#{message}"
       ```
-1. What does the program do if the response comes back with a status code of 200?
-    - Answer: 
-1. What does the program do if the response does not come back with a status code of 200?
-    - Answer: 
+
+    response = HTTParty.post(url)
+
+    if response["ok"] != true || response.code != 200
+    raise SlackError(("There was an error posting this message: #{response["error"]}"))
+    else puts "Your message has been sent successfully!" end
+
+    return response
+    end
+
+        # Copy and paste your answer above this comment
+        ```
+
+1.  What does the program do if the response comes back with a status code of 200?
+    - Answer: if the response is not true or equal to 200
```

Comment:
Hm, this answer doesn't quite make sense considering the question.

The program makes a request to send a message... and a response eventually comes back from the server, back to the program. A response always has a status code. In the case that the response has a status code of 200, what does the program do?

This question is asking what happens when the response comes back with a status code of 200. Your answer should be talking about the program's logic of what happens after that! What does a status code of 200 mean?